opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -66,6 +66,7 @@
 #include <set>
 #include <memory>
 #include <iostream>
+#include "Tables/VFPProdTable.hpp"
```

Comment:
The include should be of the same form as the table includes above (and with them).